Backend/tests/test_agent_integration.py

```
@@ -17,18 +18,15 @@ def test_agent_context():
-            print(f"Has profile: {bool(data.get('profile'))}")
-            return True
-        else:
-            print(f"❌ Context endpoint failed: {response.text}")
-            return False
+        assert response.status_code == 200, f"Context endpoint failed: {response.text}"
+        data = response.json()
+        print("✅ Context endpoint working")
+        print(f"Current week: {data.get('current_week', 'N/A')}")
+        print(f"Has profile: {bool(data.get('profile'))}")
+        return True
     except Exception as e:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)